src/main/java/com/diablo3CharViewer/GetApiAccount.java

```diff
@@ -0,0 +1,50 @@
+package com.diablo3CharViewer;
+
+public class GetApiAccount {
+
+    private String region = "eu";
+    private String locale = "pl_PL";
+    private String account;
+
+    private String requestURL;
+
+    //mozna ustawic dla innych regionow
```

Comment:
Do you want to use different regions in your app?

Reply:
only eu region